keystore/src/connection/platform/generic/migrations/V12__unique_id_constraints.sql

```diff
@@ -65,3 +65,17 @@ SELECT lower(sha256_blob(psk_id)), psk_id, psk FROM mls_psk_bundles;
 DROP TABLE mls_psk_bundles;
 
 ALTER TABLE mls_psk_bundles_new RENAME TO mls_psk_bundles;
+
+---- mls_groups ----
+CREATE TABLE mls_groups_new (
+    id_hex TEXT unique,
+    parent_id BLOB,
+    state BLOB
+);
+
+INSERT INTO mls_groups_new (id_hex, parent_id, state)
+SELECT lower(hex(id)), parent_id, state FROM mls_groups;
+
+DROP TABLE mls_groups;
```

Comment:
This commit references `mls_group` table in its message, but the table is called `mls_groups`.